set.mm

```diff
@@ -536981,6 +536982,29 @@ standard proof of the implication in modal logic (B5 ` => ` 4).  Its dual
   bj-wnfenf $p |- ( ( E. x ph -> A. x ps ) -> A. x ( E. x ph -> ps ) ) $=
     ( wex wal wi bj-wnf1 bj-19.21bit sylg ) ACDZBCEFZKJBFCABCGJBCHI $.
 
+  $( Equivalent form of the axiom of substitution ~ bj-ax12 .  Although both
+     sides need a DV condition on ` x , t ` (or as in ~ bj-ax12v3 on
+     ` t , ph ` ) to hold, their equivalence holds without DV conditions.  The
+     forward implication is proved in modal (K4) while the reverse implication
```

Comment:
So `K4` and `T5` are a numbering of the modal logic axioms here?

Reply:
Yes, this is standard terminology in modal logic. Axiom K is ax-4, Axiom T is spec, Axiom 4 is $\forall \to \forall\forall$, and Axiom 5 is $\exists \to \forall \exists$.  I updated some comments in set.mm in recent years, to indicate these correspondences (search "modal" in comments or labels of the main part of set.mm).

The modal logics (K4) and (T5) are the logics using these axioms on top of propositional calculus and the generalization rule.

This is also summarized in the section "Modal logic" in my mathbox.